.eslintrc.js

```diff
@@ -1,51 +1,41 @@
 module.exports = {
-	"env": {
-		"commonjs": true,
-		"es6": true,
-		"node": true
-	},
-	"extends": "eslint:recommended",
-	"overrides": [
-		{
-			"files": ["**/*.ts"],
-			"parser": "@typescript-eslint/parser",
-			"extends": [
-				"eslint:recommended",
-				"plugin:@typescript-eslint/eslint-recommended",
-				"plugin:@typescript-eslint/recommended"
-			],
-			"plugins": ["@typescript-eslint"],
-			"rules": {
-				"@typescript-eslint/camelcase": "off",
-				"@typescript-eslint/no-this-alias": "off",
-				"@typescript-eslint/no-var-requires": "off",
-				"@typescript-eslint/no-explicit-any": "off"
-			}
-		}
-	],
-	"globals": {
-		"Atomics": "readonly",
-		"SharedArrayBuffer": "readonly"
-	},
-	"parserOptions": {
-		"ecmaVersion": 2018
-	},
-	"rules": {
-		"indent": [
-			"error",
-			"tab"
-		],
-		"linebreak-style": [
-			"error",
-			"unix"
-		],
-		"quotes": [
-			"error",
-			"double"
-		],
-		"semi": [
-			"error",
-			"always"
-		]
-	}
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    sourceType: "module",
+  },
+  env: {
+    commonjs: true,
+    es6: true,
+    node: true,
+  },
+  plugins: ["@typescript-eslint/eslint-plugin"],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+  ],
+  rules: {
+    "@typescript-eslint/interface-name-prefix": "off",
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-unused-vars": "off",
```

Comment:
Why did we turn these off?

Reply:
`@typescript-eslint/no-explicit-any` is I believe from the previous eslint. Everything else
is based on [this](https://github.com/nestjs/nest/blob/master/.eslintrc.js) as suggested in https://github.com/dynamoose/dynamoose/issues/896. 

From my standpoint of view:

- interface-name-prefix: I really don't see reason in prefixing interfaces with I
- explicit-function-return-type: this is usually just an extra overhead for the developer from my experience Mainly considering typescript is really good with inferring return types now.
- no-explicit-any: I am personally not a fan of this but I think turning this on would slow down and make this PR big

Reply:
You do have `@typescript-eslint/no-explicit-any` twice in here tho.

`interface-name-prefix` also didn't cause any problems before. Like we didn't have that disabled in the previous version and everything was working. What changed between last version and this version that required setting that to off?

`explicit-function-return-type`. I think that should be a separate pull request, and I'm not so sure I'd approve it. I don't see the harm in it as of right now, and as a newbie to TS, it does help get the syntax down and get in the habit of declaring return types.

`no-explicit-any`. Agreed. Separate PR would be good. However, it's duplicated twice in the rules. So we need to remove one of them.